ldm/models/diffusion/shared_invokeai_diffusion.py

```diff
@@ -102,8 +102,11 @@ def apply_standard_conditioning(self, x, sigma, unconditioning, conditioning):
         x_twice = torch.cat([x] * 2)
         sigma_twice = torch.cat([sigma] * 2)
         both_conditionings = torch.cat([unconditioning, conditioning])
-        unconditioned_next_x, conditioned_next_x = self.model_forward_callback(x_twice, sigma_twice,
-                                                                               both_conditionings).chunk(2)
+        both_results = self.model_forward_callback(x_twice, sigma_twice, both_conditionings)
+        unconditioned_next_x, conditioned_next_x = both_results.chunk(2)
+        if conditioned_next_x.device.type == 'mps':
+            # prevent a result filled with zeros. seems to be a torch bug.
+            conditioned_next_x = conditioned_next_x.clone()
```

Comment:
it probably makes sense to file a bug with pytorch for this, if i can figure out a repro. simply doing 
```
import torch
a = torch.rand(2, 4, 64, 64).to('mps')
left, right = a.chunk(2)
left - right
```
in a python console is not sufficient to reproduce. 

at a guess, the `unet.forward(..).sample` called by `model_forward_callback` is returning a view, and then running `chunk(2)` on that view returns yet another view, and then subtracting a view from a view of the same underlying data triggers some weird bug that makes everything zero.

Reply:
the clone is only 1x4x64x64 floats, and it only happens once per step, so it's not a big deal, but it would be nice if we didnt' have to.

Reply:
That is so strange. I haven't seen this in the diffusers pipelines at all, and I don't know if it was something your other implementation here had been doing.

In any case, I agree that one clone per step is an acceptable workaround.

Reply:
resolution here:
- https://github.com/invoke-ai/InvokeAI/pull/1583#issuecomment-1374201383
- https://github.com/invoke-ai/InvokeAI/pull/1583#issuecomment-1374202053

workaround will stay in place until the next PyTorch release.